content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -53,11 +53,16 @@ Support for the Topology Manager requires `TopologyManager` [feature gate](/docs
 
 The Topology Manager currently:
 
- - Works on Nodes with the `static` CPU Manager Policy enabled. See [control CPU Management Policies](/docs/tasks/administer-cluster/cpu-management-policies/)
- - Works on Pods making CPU requests or Device requests via extended resources
+ - Aligns Pods of all QoS classes.
+ - Aligns the requested resources that Hint Provider provides topology hints for.
 
 If these conditions are met, Topology Manager will align the requested resources.
 
+Note: 

```

Comment:
You can use a [note shortcode](https://kubernetes.io/docs/contribute/style/style-guide/#note) here.

Reply:
Updated :)